src/mirage/fixtures.js

```diff
@@ -749,8 +749,11 @@ export const serviceRoutes = [
   {
```

Comment:
A suggest you add one more tcp route in here

Reply:
done